components/properties/src/provider.rs

```diff
@@ -27,7 +27,7 @@ use zerofrom::ZeroFrom;
 )]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[non_exhaustive]
-pub enum UnicodePropertyV1<'data> {
+pub enum PropertyUnicodeSetV1<'data> {
```

Comment:
I think this should be called `CodePointSet`, not `UnicodeSet`. @echeran @Manishearth ?

Reply:
Correct. We shouldn't have `Unicode` in those names right now.

(@andrewpollack this wasn't clear in the issue yet, but CodePoint is for char mappings, Unicode is for string mappings, currently we only support string mappings)

Reply:
er. yes. that.

Reply:
Got it! I went ahead and renamed all `UnicodeSet -> CodePointSet` to make this change consistent across the board

Reply:
> CodePoint is for char mappings

Shouldn't we use `ScalarValue` for `char`? That said, the set does support code points (i.e. surrogates are allowed in the set and you can query for then with `contains_u32()`).

Reply:
> > CodePoint is for char mappings
> 
> Shouldn't we use `ScalarValue` for `char`?

I think a more precise way of restating the intent of that phrasing is "CodePoint is for code point mappings".  In programming language contexts, "code point" and "character" are very distinct things (and in Rust in particular, there is an even more specific meaning to "character").  But -- it may just be my perception -- in some Unicode contexts, there is sometimes a casual interchangeability of the terms when referring to a code point.  I think that's what's going on here.  It's not the Rust interpretation of `char` as a type representing Unicode scalar value.

So I think the naming is fine as-is.